util/config/config.go

```diff
@@ -176,6 +179,14 @@ func Init() error {
 		Parameters.BeneficiaryAddr = BeneficiaryAddr
 	}
 
+	if len(SeedList) > 0 {
+		Parameters.SeedList = strings.Split(SeedList, ",")
+	}
+
+	if len(GenesisBlockProposer) > 0 {
```

Comment:
@yilunzhang  GenesisBlockProposer check should not only >0, but also should equal len(pubKey.Bytes())

Reply:
This is checked in some later commit